src/pages/index.js

```diff
@@ -99,7 +84,7 @@ api
     );
     cardSection.renderItems();
   })
-  .catch((err) => console.error(err));
+  .catch(console.err);
```

Comment:
`  .catch(console.err); `  should be `.catch(console.error);` because in `log` there is method `error` for logging errors